packages/amplify_analytics_pinpoint/android/src/main/kotlin/com/amazonaws/amplify/amplify_analytics_pinpoint/AmplifyAnalyticsBridge.kt

```diff
@@ -34,11 +34,11 @@ class AmplifyAnalyticsBridge {
             try {
                 Amplify.addPlugin(AWSPinpointAnalyticsPlugin(context as Application?))
                 LOG.info("Added AnalyticsPinpoint plugin")
+                flutterResult.success(null)
             } catch (e: Exception) {
```

Comment:
Do we need to handle flutterResult.error(...) here?

Reply:
@fjnoyp?

Reply:
For this case no.  There are other places in our code where that would be necessary such as if we were to have additional code after this try catch.

Reply:
That's okay as there is no code below and every code path is calling flutterResult.